res/css/views/rooms/_EventBubbleTile.scss

```diff
@@ -150,7 +150,8 @@ limitations under the License.
         }
 
         .mx_MessageActionBar {
-            right: -100px; // to make sure it doesn't overflow to the left or cover sender profile
+            right: initial;
```

Comment:
It was addressed with: https://github.com/matrix-org/matrix-react-sdk/pull/8398/commits/03b5f20acde8fccd7c69dd28b6a0eb76b030fd87#diff-11808ff6468e6efa5982f414119fcc8a3a3eea4090b1a4ebda8ed6eae732418aR154

It is required to avoid the bar from being compressed. https://github.com/matrix-org/matrix-react-sdk/pull/8398#discussion_r871646225